File: PHP/updateComment.php
<?php
    
    $auth_token = '59550a0e2b1a864a31bef962363e029f'; //your_auth_token
    $org_id=652853630; //your_organization_id
    $ticket_id="215666000000074114";
    $comment_id="215666000000166017";
    
    $comment_data=array(
        "content"=>"This is changed sample comment content"
    );
    
    $headers=array(
            "Authorization: $auth_token",
            "orgId: $org_id",
            "contentType: application/json; charset=utf-8",
    );
    
    if($ticket_id && $comment_id){
        $url="https://desk.zoho.com/api/v1/tickets/$ticket_id/comments/$comment_id";

        $comment_data=(gettype($comment_data)==="array")? json_encode($comment_data):$comment_data;

        $ch= curl_init($url);
        curl_setopt($ch,CURLOPT_HTTPHEADER,$headers);
        curl_setopt($ch,CURLOPT_RETURNTRANSFER,TRUE);
        curl_setopt($ch,CURLOPT_CUSTOMREQUEST,"PATCH");
        curl_setopt($ch, CURLOPT_POSTFIELDS,$comment_data); //convert ticket data array to json

        $response= curl_exec($ch);
        $info= curl_getinfo($ch);

        if($info['http_code']==200){
            echo "<h2>Request Successful, Response:</h2> <br>";
            echo $response;
        }
        else{
            echo "Request not successful. Response code : ".$info['http_code']." <br>";
            echo "Response : $response";
        }

        curl_close($ch);
    }
    else{
        echo "ERROR : ";
        echo (!$ticket_id)?"Ticket ID ":"";
        echo (!$comment_id)?" Comment ID ":"";
        echo " is missing";
    }   
    
?>

File: PYTHON/updateComment.py
import requests,json

auth_token="59550a0e2b1a864a31bef962363e029f" #YOUR_AUTH_TOKEN
org_id="652853630" #YOUR_ORGANISATION_ID

ticket_id="215666000000137001"; #TICKET_ID_OF_COMMENT
comment_id="215666000000173005"; #COMMENT_ID_TO_UPDATE

comment_data={
    "content":"Changed content by python"
}

headers={
    "Authorization":auth_token,
    "orgId":org_id,
    "contentType": "application/json; charset=utf-8"
}

if ticket_id and comment_id:

    request=requests.patch('https://desk.zoho.com/api/v1/tickets/'+ticket_id+'/comments/'+comment_id, headers=headers,data=json.dumps(comment_data))

    if request.status_code == 200:
        print "Request Successful,Response:"
        print request.content
    else:
        print "Request not successful,Response code ",request.status_code," \nResponse : ",request.content;

else:
    print "ERROR : ",
    if not ticket_id: print "Ticket ID ",
    if not comment_id: print "Comment ID ",
    print "is missing."